### PR TITLE
Fix: Only parse json from stdout

### LINKED ISSUE #53: CombinedOutput causes unmarshal error
Some functions such as `ListMembersOfTeam` will error if the keybase cli has any warnings, as the function tries to unmarshal both stdout and stderr. Here is an example payload that failed unmarshaling:
```
2020-02-12T16:04:47.296724-05:00 ▶ [WARN keybase versionfix.go:236] 001 KBFS needs to restart; running version 5.2.0, but 5.3.0-20200212090128+1e4b858d80 installed.
{"result": "..."}
```

### PR DESCRIPTION
**Pull request by marceloneil**
Fixes #53

### CODE REVIEW

File: kbchat/team.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ func (a *API) ListMembersOfTeam(teamName string) (res keybase1.TeamMembersDetail
 	apiInput := fmt.Sprintf(`{"method": "list-team-memberships", "params": {"options": {"team": "%s"}}}`, teamName)
 	cmd := a.runOpts.Command("team", "api")
 	cmd.Stdin = strings.NewReader(apiInput)
-	bytes, err := cmd.CombinedOutput()
+	bytes, err := cmd.Output()
 	if err != nil {
 		return res, APIError{err}
```
Comment (joshblum) [inline kbchat/team.go]: can you read from stderr on error and log?

Comment (marceloneil) [reply kbchat/team.go]: Should I log stderr if it isn't empty, or just if there is an `err`?

Comment (joshblum) [reply kbchat/team.go]: always logging is fine, thanks!

Comment (marceloneil) [reply kbchat/team.go]: I log stderr for the methods that I modified, however this isn't don't elsewhere and probably should be.

**Pull request merged**